src/Routes.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
There is no need for this component to be a class based component @Manorlds-Eaglespark

Reply:
> There is no need for this component to be a class-based component @Manorlds-Eaglespark

Thanks, @Yiga-fred. The implementation has been changed from class-based to function based.